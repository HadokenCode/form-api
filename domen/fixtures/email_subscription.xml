<form id="41ca5e09-3ce2-4094-b108-3ecc257c6fa4" title="Email subscription"
      action="http://localhost:8080/api/v1/41ca5e09-3ce2-4094-b108-3ecc257c6fa4" method="post"
      enctype="application/x-www-form-urlencoded">
    <input id="41ca5e09-3ce2-4094-b108-3ecc257c6fa4_email" name="email" type="email" title="Email" maxlength="64"
           required="1"/>
    <input id="41ca5e09-3ce2-4094-b108-3ecc257c6fa4__redirect" name="_redirect" type="hidden"
           value="https://kamil.samigullin.info/"/>
</form>
