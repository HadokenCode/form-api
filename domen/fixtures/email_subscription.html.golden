<form id="41ca5e09-3ce2-4094-b108-3ecc257c6fa4" title="Email subscription" action="http://form-api.dev/api/v1/41ca5e09-3ce2-4094-b108-3ecc257c6fa4" method="post" enctype="application/x-www-form-urlencoded"><label for="41ca5e09-3ce2-4094-b108-3ecc257c6fa4_email">Email</label><input id="41ca5e09-3ce2-4094-b108-3ecc257c6fa4_email" name="email" type="email" title="Email" maxlength="64" required><input id="41ca5e09-3ce2-4094-b108-3ecc257c6fa4__redirect" name="_redirect" type="hidden" value="https://kamil.samigullin.info/"><input type="submit"></form>
